quickstarts/apache-flink/config.yml

```diff
@@ -0,0 +1,44 @@
+
+
+slug: apache-flink
+description: |
+  ## Comprehensive monitoring quickstart for Apache Flink
+  New Relic detects your Apache Flink metrics like job manager (checkpoints, direct, mapped, metaspace memory usage),
+  task manager (shuffle netty, operators), heap memory usage and more.
+
+  ## Why monitor Apache Flink?
+  An ideal Apache Flink Performance Monitoring provides an easy way for effective monitoring and alerting of your Flink jobs.
```

Comment:
I think we need a noun here for this part of the sentence:
`An ideal Apache Flink Performance Monitoring provides`

Additionally let's lower case "Performance Monitoring"

For example:

An ideal Apache Flink performance monitor provides
Ideal Apache Flink performance monitoring provides

Reply:
Going to approve for now, and address this in a follow up